portfolio/src/main/webapp/script.js

```diff
@@ -194,7 +222,12 @@ function makeMap() {
 let markers = [];
 let marker;
 
-function addPin(comment, time, i) {
+/** 
+ * Creates and drops a pin and info window for a given comment
+ * @param {comment} The comment to be displayed.
+ * @param {comment} The time delay for animating the pin drop.
```

Comment:
Should this be number?

Reply:
It should! Thanks for catching that, changing the second param to "time".